deploy/helm/kubecf/assets/operations/stacks.yaml

```diff
@@ -0,0 +1,178 @@
+{{- $_ := include "_config.lookup" (list $ "releases") }}
+{{- range $release_name, $release := $.kubecf.retval }}
+  {{- if $release.buildpack }}
+    {{- /* buildpack release conditions are either true or false, not complex expressions */}}
```

Comment:
It _could_ be a complex condition, but the condition is set by the logic in `_stacks.update`. It is initially set to `$stack.enabled`, which is a boolean that indicates if the stack is included in `install_stacks` or not.

Then the condition may be changed to `false` if the buildpack is not included in the `$stack.install_buildpacks` list.

The comment just explains why we can use the value directly as a boolean and don't need to run it through `_config.condition`. We could thought, but it would be pointless. Does this need a better explanation?

Reply:
I think something like "buildpack release conditions are set in `_stacks.update` to either `true` or `false`" would be enough (the key being, it's not something somebody could go tweak).

Reply:
Done in be2fe37